src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Http/ClientCredentialTypeTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// 
+
+using System;
+using System.Net;
+using System.Security.Principal;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+
+using Infrastructure.Common;
+
+using Xunit;
+
+public static class Http_ClientCredentialTypeTests
+{
+    [Fact]
+    [OuterLoop]
+    public static void DigestAuthentication_Echo_RoundTrips_String()
+    {
```

Comment:
Nit: Recommend adding to method name "_No_Domain" or something to clue others into the fact no domain is required.  At some point we'll add [ConditionalFact] around any auth tests that do require it, and it will help keep them straight.